msc_pygeoapi/loader/aqhi_realtime.py

```diff
@@ -0,0 +1,383 @@
+# =================================================================
+#
+# Author: Etienne Pelletier <etienne.pelletier@canada.ca>
+#         Felix Laframboise <felix.laframboise@canada.ca>
+# Copyright (c) 2020 Etienne Pelletier
+#               2020 Felix Laframboise
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation
+# files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use,
+# copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following
+# conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# =================================================================
+
+from datetime import datetime
+import json
+import logging
+import os
+from pathlib import Path
+
+import click
+from elasticsearch import logger as elastic_logger
+from parse import parse
+
+from msc_pygeoapi import cli_options
+from msc_pygeoapi.connector.elasticsearch_ import ElasticsearchConnector
+from msc_pygeoapi.env import (
+    MSC_PYGEOAPI_LOGGING_LOGLEVEL
+)
+from msc_pygeoapi.loader.base import BaseLoader
+from msc_pygeoapi.util import (
+    configure_es_connection,
+    check_es_indexes_to_delete,
+)
+
+LOGGER = logging.getLogger(__name__)
+elastic_logger.setLevel(getattr(logging, MSC_PYGEOAPI_LOGGING_LOGLEVEL))
+
+# cleanup settings
+DAYS_TO_KEEP = 3
+
+# index settings
+INDEX_BASENAME = 'aqhi_realtime.{}.'
```

Comment:
` INDEX_BASENAME = 'aqhi-realtime-{}.'`

Reply:
Is it not supposed to be `INDEX_BASENAME = 'aqhi-{}-realtime.'` following comments above?

Reply:
Ah, yes, `aqhi-{}-realtime.'`